.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
```

Comment:
I'm not 100% we actually use this github action as CI seemed to run before I added this action? But I see it in other datasources so I sort of assume it's needed!

Reply:
hmm yeah this seems to add an additional CI step that seems very similar to the setup we already have in drone, should I just remove maybe?